packages/common/src/stream.ts

```diff
@@ -91,7 +91,7 @@ export interface StreamMetadata {
   schema?: string
   tags?: Array<string>
   forbidControllerChange?: boolean
-
+  model?: StreamID
```

Comment:
this already exists in the `develop` branch, please merge in develop branch and tag me to take another look after you do and the tests are passing